scenes/game/local_game.gd

```diff
@@ -2382,6 +2405,35 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			var close_amount = abs(performing_start - new_location)
 			local_conditions.fully_closed = close_amount == effect['amount']
 			_append_log("%s Close %s - Moved from %s to %s." % [performing_player.name, str(effect['amount']), str(previous_location), str(new_location)])
+		"copy_other_hit_effect":
+			var card = null
+			if performing_player == active_strike.initiator:
```

Comment:
You should use active_strike.get_player_card(performing_player : Player) -> GameCard